server/schema/contests/index.js

```diff
@@ -1,11 +1,21 @@
+const ajv = require('../index')
 const createContestSchema = require('./createContest')
+const updateContestSchema = require('./updateContest')
 const addModeratorSchema = require('./addModerator')
 const removeModeratorSchema = require('./removeModerator')
-const updateContestSchema = require('./updateContest')
+const addGroupSchema = require('./addGroup')
+const removeGroupSchema = require('./removeGroup')
+
+ajv.addFormat('dateTimeFormat', (dateTimeString) => {
+  const timestamp = new Date(dateTimeString)
+  return timestamp instanceof Object && typeof timestamp === 'object'
```

Comment:
This validator is also accepting invalid strings like "a", "b", etc.
So, I suggest this modifying this as :
```suggestion
  return !isNaN(Date.parse(dateTimeString));
```

Reply:
You can try running this in console :
```
const timestamp = new Date("a")
console.log(timestamp instanceof Object && typeof timestamp === 'object');
```

Reply:
@NBNARADHYA , what is your review ?

Reply:
```suggestion
  Object.prototype.toString.call(timestamp) === '[object Date]'
```

Reply:
return ?

Reply:
Try running this in console :
```
var timestamp = new Date("a")
console.log(Object.prototype.toString.call(timestamp) === '[object Date]');
```

Reply:
Actually, the prototype would same, even for invalid dates, i guess. Better to use Date.parse(), it is working properly.